.github/workflows/auto-merge.yml

```
@@ -0,0 +1,141 @@
+          # Force merge PRs that are approved but not mergeable (due to checks)
+          jq -r '.[] | @base64' stuck-prs.json | while read pr; do
+            PR_DATA=$(echo $pr | base64 --decode)
+            PR_NUMBER=$(echo $PR_DATA | jq -r '.number')
+            PR_TITLE=$(echo $PR_DATA | jq -r '.title')
+            MERGEABLE=$(echo $PR_DATA | jq -r '.mergeable')
+            REVIEW_DECISION=$(echo $PR_DATA | jq -r '.reviewDecision')
+            
+            # If approved but not mergeable, force merge
+            if [ "$REVIEW_DECISION" == "APPROVED" ] && [ "$MERGEABLE" != "MERGEABLE" ]; then
+              echo "Force merging stuck PR #$PR_NUMBER: $PR_TITLE"
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)